src/elkan5e.mjs

```
@@ -64,4 +66,26 @@ Hooks.on("dnd5e.postUseActivity", (activity, usageConfig, results) => {
+
+Hooks.on("deleteItem", async (item, options, userId) => {
+    delayedItem(item);
+});
+/**
+ * Handle end of turn activities.
+ * @param {object} combatant - The combatant whose turn ended.
+ * @param {object} turn - The turn data.
+ */
+Hooks.on("combatTurnChange", (combat, prior, current) => {
+    // Add your end of turn logic here
+    let lastTurnActor = combat.combatants.get(prior.combatantId).actor
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'lastTurnActor' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)